packages/pipeline/samples/typescript/src/pipeline.ts

```diff
@@ -1,19 +1,27 @@
 /*! Copyright [Amazon.com](http://amazon.com/), Inc. or its affiliates. All Rights Reserved.
 SPDX-License-Identifier: Apache-2.0 */
 import { PDKNag } from 'aws-prototyping-sdk/pdk-nag';
+import { PipelineUtils } from 'aws-prototyping-sdk/pipeline';
 import { ApplicationStage } from './application-stage';
 import { PipelineStack } from './pipeline-stack';
 
 const app = PDKNag.app();
 
-const pipelineStack = new PipelineStack(app, 'PipelineStack', {
+const branchPrefix =
+  process.env.BRANCH &&
+  process.env.BRANCH !==
+    (app.node.tryGetContext('defaultBranchName') || PipelineUtils.defaultBranchName)
+    ? PipelineUtils.normalizeBranchName(process.env.BRANCH) + '-'
+    : '';
```

Comment:
It might be nicer to abstract away the environment variable from users, and provide some helpers like `PDKPipeline.getBranchPrefix(app)` and `PDKPipeline.isDefaultBranch(app)`?